connect/src/main/java/com/stripe/android/connect/AccountOnboardingView.kt

```diff
@@ -6,50 +6,51 @@ import android.util.AttributeSet
 import androidx.annotation.RestrictTo
 import androidx.core.content.withStyledAttributes
 import com.stripe.android.connect.webview.StripeConnectWebViewContainer
-import com.stripe.android.connect.webview.StripeConnectWebViewContainerImpl
 import com.stripe.android.connect.webview.serialization.SetOnExit
 import com.stripe.android.connect.webview.serialization.SetterFunctionCalledMessage
 import dev.drewhamilton.poko.Poko
 import kotlinx.parcelize.Parcelize
 
 @PrivateBetaConnectSDK
-class AccountOnboardingView private constructor(
+internal class AccountOnboardingView internal constructor(
     context: Context,
-    attrs: AttributeSet?,
-    defStyleAttr: Int,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0,
+    embeddedComponentManager: EmbeddedComponentManager?,
```

Comment:
👀 Note how the new params are the same as those passed to `webViewContainerBehavior` previously.